content/millennium/r4/diagnostic/observation.md

```diff
@@ -246,9 +246,9 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
 * Both read and write scopes are required.
-* Updates on patient and category are not supported. Updating a value to a different value type is not supported.
+* Updates on patient and category are not supported.
```

Comment:
Can you clarify why this changed? I'm pretty sure this is still accurate.

Reply:
@joeratt we fixed that for labs and support updates to change to different value types

Reply:
Do vitals updates support changing type, though?

Reply:
@joeratt no they don’t. That’s what I get for looking at this on my phone 😂

Reply:
I see the "labs only" limitation called out here: https://github.com/cerner/fhir.cerner.com/pull/675/files#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR640

I'm not sure why we have 2 different sections communicating the same info. @maruthiallam  - can you help make sure they're in-sync, please?

Reply:
The idea is updating value type to the same value type vs updating to other value type.  I have updated it to look like the  below screenshot. Let me know if it's still confusing.
<img width="573" alt="Screen Shot 2021-06-18 at 9 24 55 AM" src="https://user-images.githubusercontent.com/18075594/122576309-2ba52300-d017-11eb-8fdf-6cb37d418bcd.png">

Reply:
nit: Can we put the labs one together and have something like
Laboratory supports `valueQuantity`, `valueCodeableConcept`, `valueString`.
Laboratory also, supports updating observations with 'valueDateTime' to 'valueDateTime' type.
Vital Signs support only updating observations with `valueQuantity` to `valueQuantity` type

Any thoughts?

Reply:
Discussed offline and decided that it better to have it based on type.

Reply:
The exception is valueDateTime cannot be updated to **another** value type...

Reply:
<img width="656" alt="Screen Shot 2021-06-18 at 10 55 02 AM" src="https://user-images.githubusercontent.com/18075594/122587627-ac6a1c00-d023-11eb-909b-cd66e70b7f62.png">